webpack.mix.js

```diff
@@ -11,5 +10,28 @@ const mix = require('laravel-mix');
  |
  */
 
-mix.js('resources/js/app.js', 'public/js')
-   .sass('resources/sass/app.scss', 'public/css');
+mix.js('resources/js/materialize_custom.js', 'public/js/materialize.js') // We use our custom materialize JS
+   .js('resources/js/cookieconsent-initialize.js', 'public/js/')
+   .js('resources/js/register_helper.js', 'public/js/')
+   .js('resources/js/site.js', 'public/js/')
+   .js('resources/js/csrf.js', 'public/js/')
+   // We have to copy already minimized JS
+   .copy('resources/js/cookieconsent.min.js', 'public/js/') // TODO: see #223
+   .copy([
+      'node_modules/moment/min/moment.min.js',
+      'node_modules/jquery/dist/jquery.min.js',
+      'node_modules/tabulator-tables/dist/js/tabulator.min.js'
+   ], 'public/js/')
+   // Compile SASS
+   .sass('resources/sass/materialize.scss', 'public/css/') // We store it locally, but probably could be imported
```

Comment:
Yes, but unfortunately there are some custom, overwritten values now in scss - we need to overwrite those in a separate file. Might create an issue for that.

Reply:
Do you mean that you modified the materialize.scss?

Reply:
Yes, eg. with colors. I will try to separate those in this issue.